boards/mips-malta/malta.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016, Imagination Technologies Limited and/or its
+ *                 affiliated group companies.
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+
+#include <stdio.h>
+
+#include "periph/uart.h"
+
+#define MIPS_MALTA_ADDR 0xbf000500
+#define MIPS_MALTA_VAL_RST 0x42
```

Comment:
It's a matter of taste, but I would appreciate parens around all these macro definitions.